packages/api-explorer/style/main.scss

```diff
@@ -0,0 +1,2 @@
+@import '../api-explorer.css';
+@import './schema-form.scss';
```

Comment:
@domharrington I think we ought to keep the main `api-explorer.css` file as-is, in case other repos are still referencing it? (At least until further thought is given to our cross-repository CSS architecture!) But for now I've "combined" the two separate partials with SCSS `@import`s! Ça va, oui?

Reply:
No other repos are referencing it apart from ReadMe's main codebase. You're good to merge them together!

Reply:
you got it